tests/model/templates/tester-response-template.model.spec.ts

```diff
@@ -14,4 +26,22 @@ describe('TesterResponseTemplate class', () => {
       expect(template.teamChosenType).toBe(TYPE_DOCUMENTATION_BUG);
     });
   });
+  describe('testerResponse and teamResponse fields', () => {
+    it('parses the testerResponse and teamResponse fields correctly from the GithubComment', () => {
+      const template = new TesterResponseTemplate([TEAM_RESPONSE_MULTIPLE_DISAGREEMENT]);
+
+      expect(template.teamResponse.content).toBe(EXPECTED_TEAM_RESPONSE_CONTENT);
+      expect(template.teamResponse.header.toString()).toBe(EXPECTED_TEAM_RESPONSE_HEADER);
+
+      expect(template.testerResponse.testerResponses[0].title).toBe(ISSUE_SEVERITY_TITLE);
```

Comment:
How about defining something like `firstResponse = template.testerResponse.testerResponses[0]`, just to prevent repetition?